packages/dso-toolkit/libs/bootstrap/_scaffolding.scss

```diff
@@ -21,18 +21,20 @@
 
 // Body reset
 
-html {
-  font-size: 10px;
-  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
-}
-
-body {
-  font-family: $font-family-base;
-  font-size: $font-size-base;
-  line-height: $line-height-base;
-  color: $text-color;
-  background-color: $body-bg;
-}
+// html {
+//   font-size: 10px;
+//   -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+// }
+// -> Moved to @dso-toolkit/styling/scaffolding/typography
+
+// body {
+//   font-family: $font-family-base;
+//   font-size: $font-size-base;
+//   line-height: $line-height-base;
+//   color: $text-color;
+//   background-color: $body-bg;
+// }
+// -> Moved to @dso-toolkit/styling/scaffolding/typography
 
```

Comment:
@daverhelst Vraagje voor mijn begrip: laten we deze uitgecommentarieerde css bewust staan? Of laten we het voorlopig staan en met een opschoonactie aan het einde van dit project weghalen?

Reply:
Voor nu laten we dit staan, zodat we weten wat er precies is overgezet (en misschien is weggehaald, want niet gebruikt of overbodig)

Reply:
Okidoki